### PR TITLE
sync fork with upstream

### PR DESCRIPTION
**Pull request by HassanBahati**
@amkayondo in this pr, I am syncing this fork with the updates from the [upstream](https://github.com/airqo-platform/AirQo-frontend)

### CODE REVIEW

**Comment by amkayondo** [comment]
lets wait for a review from @123MwanjeMike

**Review by 123MwanjeMike — APPROVED** [review]
Looks Good To Me 
Thanks @HassanBahati

**Pull request merged**
